src/pages/dataTableUtilities/getPageInitialiser.js

```diff
@@ -377,7 +374,7 @@ const supplierRequisitionInitialiser = requisition => {
     modalKey: '',
     hasSelection: false,
     modalValue: null,
-    showAll: !usingPrograms,
+    showAll: usingPrograms,
```

Comment:
I think the intention was to still have `showAll` to be false when using a program for a NON-finalised requisition. This makes it easier when for example, you go to first make the requisition, you can see only the items you *should* order.. but when you view an old, finalised, requisition, you display all the items so none are hiding. So I think we should still use `filteredData` - but only when the `status !== "finalised"` -- make sense?

Reply:
Totally! Yes, I've got it. Now in the filter I am checking the status and `showAll` only will be true when the using a program that is finalised. Let me know what you think about it. Thanks!